packages/dds/tree/src/test/feature-libraries/optional-field/optionalFieldUtils.ts

```diff
@@ -152,7 +152,7 @@ export const Change = {
 		const changeset: Mutable<OptionalChangeset> = { moves, childChanges };
 		for (const changeLike of changes) {
 			if ("type" in changeLike === false) {
-				const change = changeLike as OptionalChangeset;
+				const change = changeLike;
```

Comment:
improved narrowing made this `as` cast a no-op, so it had to be removed as per our compiler settings.